File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Insert height: \n");
    }
    while (height < 1 || height > 8);

    for (int row = 1; row <= height; row++)
    {
        for (int spaces = height - row ; spaces > 0; spaces--)
        {
            printf(" ");
        }
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }
        printf("  ");
        for (int hash = 1; hash <= row; hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

while True:
    input = get_int("Insert number: ")
    input = int(input)
    if input in range(1, 9):
        break

spaces = input - 1
hash = 1

for i in range(input):
    print((" " * spaces) + ("#" * hash))
    spaces -= 1
    hash += 1

